### ISSUE TITLE
Publish on NPM

### ISSUE THREAD

**Issue by armandabric** [jamesallardice/Placeholders.js#90]
I :heart:  this package, but I can't use it on my current project :sob: Yes this project only use NPM as package manager. 

As I'm not the only one who drop the use to bower, could you publish on NPM too ? Thanks a lot! :cake: :cake: :cake:

**Comment by Ekhoo** [comment]
:+1:

**Comment by shouze** [comment]
:+1:

**Comment by shouze** [comment]
Any news about this one?

**Comment by kendagriff** [comment]
+1

**Comment by sidonaldson** [comment]
:pray:

**Comment by timbru31** [comment]
This would be awesome :)

**Comment by kroko** [comment]
+1

**Comment by Ghostavio** [comment]
please +1

**Comment by rupurt** [comment]
:+1:

**Comment by sidonaldson** [comment]
I'm confused. It does appear to be on NPM but an old version? https://www.npmjs.com/package/placeholders.js

**Comment by shouze** [comment]
ping @jamesallardice would you please push the latest release to npm?

https://www.npmjs.com/package/Placeholders.js thx! Or anyway you can design someone as repo collab to make this happen?

**Comment by export-mike** [comment]
FYI I've pushed this to npm on `npm install fork-placeholder.js`, I'll remove it when this repo is finally published.

**Comment by shouze** [comment]
@export-mike thx but @jamesallardice why you don't want to publish on npm??! This issue is 6 months old and it takes few minutes to do that and many volunteers to make this happen.

**Comment by export-mike** [comment]
Yeah I'm not going to stop a publish to npm to cause updates to my build setup, or even make me consider copying the lib into my project and have it checked in. This was a quick workaround until this issue gets resolved.
